flashinfer/gemm/kernels/tgv_gemm_cute_ext.py

```
@@ -0,0 +1,1079 @@
+           = ((8, 16), 1, 8, 8)       — Sw<3,4,3>
+  tAcc:       ((Mma_M, Mma_N), NumMma_M, NumMma_N)
+           = (((16,4), 8), 1, 1)      — fp32 in TMEM
+
+## Integration with FlashInfer
+This module is wired up as the default ``"tgv"`` backend in
+:func:`flashinfer.mm_bf16` / :func:`flashinfer.bmm_bf16` (the C++ TGV path
+is reachable by flipping ``_TGV_USE_CPP`` in ``flashinfer/gemm/gemm_base.py``).
+The 11 ``cute_ext`` tactics here mirror the 11 C++ tactics in
+``include/flashinfer/gemm/tgv_gemm_configs.h`` 1:1, so tactic ids are
+interchangeable across implementations and the same default
+(tactic 1 = 64×8 / 8 stages) is used when autotune is off.
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)